tests/test_data.yml

```diff
@@ -0,0 +1,19 @@
+crop:
+  num_rows: 2
+  percentage_share:
```

Comment:
Isn't this file auto-generated via the tests? This shouldn't be pushed if it is.